shared/config.default/tce_paths.php

```diff
@@ -31,22 +31,35 @@
 /**
  * Host URL (e.g.: "http://www.yoursite.com").
  */
-define('K_PATH_HOST', '');
+$scheme      = empty($_SERVER['HTTPS']) ? 'http://' : 'https://';
```

Comment:
These are not set on purpose for performance and security.
It also difficult to automatically account for all installation scenarios.
It think it is better to encourage to manually set the right value.